tests/subsys/input/input_longpress/src/main.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/sys/util.h"
```

Comment:
why `""` here?

Reply:
Because copy paste! :-) Actually not even sure why is this here in the first place, dropped.